.pipelines/release-e2e-job.yaml

```diff
@@ -0,0 +1,31 @@
+trigger:
+  branches:
+    include:
+    - main
+  tags: 
+    include:
+    - v*
+
+pr:
+  branches:
+    include:
+    - main
+  paths:
+    exclude:
+      - docs/*
+
+variables:
+  chart.name: osm-arc
+  chart.path: $(System.DefaultWorkingDirectory)/charts/$(chart.name)
+  image.dir: $(Build.ArtifactStagingDirectory)
+  image.artifact.name: drop
+  KUBECONFIG: $(System.DefaultWorkingDirectory)/kubeconfig.json
+  image.tag: 0.1.0
```

Comment:
We use image tags of 0.1.0 for all the charts packaged/pushed to staging ACR in the CI.

Reply:
Why is 0.1.0 used for each chart pushed to staging?

Reply:
My thinking was that it would be easier to identify which charts we use for just for testing in staging (ex: in case some of them don't get deleted properly). Another reason is that, I don't know if there's a way for the pipeline to know the exact version of the current helm chart specified in Chart.yaml (a string parse with Helm show might work)?. The only other way I can think of to get the current version is to have a variable in the UI that we would continuously update after reach release to reflect the current chart version in main - but we would have to continuously update this after every release.

Reply:
Update: I found a way to use the current version of the chart instead of hardcoding 0.1.0.